src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/CommonHealthSource.utils.ts

```diff
@@ -206,6 +282,15 @@ export const handleValidateCustomMetricForm = ({
     errors = validateAssignComponent(formData, getString, errors)
   }
 
+  if (isLogsTableIsEnabled) {
```

Comment:
isLogsTableIsEnabled
```suggestion
  if (isLogsTableEnabled) {
```

Reply:
Done